assignment-client/src/avatars/AvatarMixerSlave.cpp

```diff
@@ -212,6 +212,15 @@ void AvatarMixerSlave::broadcastAvatarData(const SharedNodePointer& node) {
     _stats.jobElapsedTime += (end - start);
 }
 
+AABox computeBubbleBox(const AvatarData& avatar, float bubbleExpansionFactor) {
+    glm::vec3 position = avatar.getClientGlobalPosition();
+    glm::vec3 scale = 2.0f * (avatar.getClientGlobalPosition()- avatar.getGlobalBoundingBoxCorner());
```

Comment:
This is incorrect because of the offset. This should just be _avatar->getGlobalBoundingBoxDimensions(); ( which needs to be exposed)